perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -380,6 +380,52 @@ public int compare(Group groupToCompare,Group groupToCompareWith) {
 			}
 		}
 
+		// remove admin roles of group
+		List<Facility> facilitiesWhereGroupIsAdmin = getGroupsManagerImpl().getFacilitiesWhereGroupIsAdmin(sess, group);
+		for (Facility facility : facilitiesWhereGroupIsAdmin) {
+			try {
+				perunBl.getFacilitiesManager().removeAdmin(sess, facility, group);
```

Comment:
Please call ``perunBl..getFacilitiesMangerBl()`` to get rid of ``PrivilegeException``. Same for other removeAdmin() calls in this.